newspaper/article.py

```diff
@@ -89,14 +89,20 @@ def __init__(self, url, title='', source_url='', config=None, **kwargs):
 
         # This article's unchanged and raw HTML
         self.html = ''
+        if html:
+            self.set_html(html)
 
         # The HTML of this article's main node (most important part)
         self.article_html = ''
 
         # Flags warning users in-case they forget to download() or parse()
         # or if they call methods out of order
         self.is_parsed = False
+
         self.is_downloaded = False
+        if self.html:
+            # User has loaded html content, does not require download restriction
+            self.is_downloaded = True
```

Comment:
If the user passes some value for `self.html`, then we want to pass the `is_downloaded` check.